### PR TITLE
Rufai Proposal Post

### PR DESCRIPTION
**Pull request by rufai**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [ ] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [ ] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [ ] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [ ] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [ ] Select a beautiful and meaningful teaser image for your blog
- [ ] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Review by huan — CHANGES_REQUESTED** [review]
Thank you very much for submitting a blog post to the Wechaty blog.

Please make sure the CI turns green for your PR so that we can review and merge it!

File: jekyll/_posts/2021-05-05-gsod-project-proposal-rufai-wechaty.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title:  "GSOD'21 project proposal for wechaty"
```
Comment (huan) [inline jekyll/_posts/2021-05-05-gsod-project-proposal-rufai-wechaty.md:2]: Please add your name at the beginning of the title: `title: "Rufai Mustapha: GSOD'21 project proposal for Wechaty"`

**Review by huan — CHANGES_REQUESTED** [review]

File: jekyll/_posts/2021-05-05-gsod-project-proposal-rufai-wechaty.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+author: rufai
+categories: article
```
Comment (huan) [inline jekyll/_posts/2021-05-05-gsod-project-proposal-rufai-wechaty.md:4]: Please use `categories: proposal` for your post.

**Comment by Rohitesh-Kumar-Jain** [comment]
Hi @rufai are you still working on this PR?

**Comment by Rohitesh-Kumar-Jain** [comment]
Hi @rufai, 

are you working on this PR? Unfortunately, the CI is still red, so we can't merge this.

**Comment by Rohitesh-Kumar-Jain** [comment]
Feel free to reopen this PR, if you want to push new commits and want to merge your proposal with us !

**Pull request closed without merging**